qm-plugins/qm-object-cache/class-qm-output-object-cache.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Output class
+ *
+ * Class QM_Output_ObjectCache
+ */
+class QM_Output_ObjectCache extends QM_Output_Html {
+
+	public function __construct( QM_Collector $collector ) {
+		parent::__construct( $collector );
+
+		add_filter( 'qm/output/menu_class', array( $this, 'admin_class' ) );
+		add_filter( 'qm/output/menus', array( $this, 'admin_menu' ), 101 );
+	}
+
+	/**
+	 * Outputs data in the footer
+	 */
+	public function output() {
+		?>
+		<div class="qm qm-non-tabular" id="<?php echo esc_attr( $this->collector->id() ); ?>">
```

Comment:
I see that this `<div>` is not closed in this method. I just want to confirm that this is OK.